### ISSUE TITLE
Support for opening more than one file at the same time

### ISSUE THREAD

**Issue by miceno** [aurora/rmate#16]
When invoking rmate with more than one file location, the script complains:

``` shell
server# rmate conf/last_httpd.conf conf/last_httpd.include 
dirname: extra operand `conf/last_httpd.include'
Try `dirname --help' for more information.
/root/bin/rmate: line 226: [: conf/last_httpd.conf: 
  binary operator expected
```

**Comment by randy3k** [comment]
#46 added this support.

**Comment by Phrogz** [comment]
+1 for this to be merged (in a way that is supported by Sublime)

**Comment by aurora** [comment]
Ok ... i think i'll have some time too look into this end of january. I am still a little unhappy about the amount of changes required ... isn't there a more simple solution?

**Comment by randy3k** [comment]
Actually, I didn't change a lot of code, however, I have to wrap a chunk of code in a function in order to dispatch it multiple times. I can leave the code inside the function un-intented if you prefer.

**Comment by aurora** [comment]
No, the indent is absolutely ok ... it's the usage of bash arrays i dislike a little bit ;) ... but i have to admit, that this is only personal taste ... ok. Let's give me some more days, let's say until end of january. I hadn't have time yet to test this. But i'll do so and if everything works finde, i'll going to merge this!

**Comment by randy3k** [comment]
Actually, I also want to know if there are any possible alternatives to bash arrays.

**Comment by aurora** [comment]
Ok, i was not able to make a decision by end of january. I am sorry about this. But the more i think about the feature, the more i agree, that it would be useful to have. So currently i am working on it, but with a little different approach as yours @randy3k . It's currently to early to show something, but i just wanted to let you know, that i am working on it ...

**Comment by randy3k** [comment]
Thanks, I am looking forward to the update.

**Comment by aurora** [comment]
This was implemented in #46.
